### DISCUSSION TITLE
repetition words & filler words not existed in transcription results

### DISCUSSION THREAD

**Discussion by jayn1985** [openai/whisper#621] [Q&A]
I tried Whisper for ASR generation these days and was shocked by the accuracy of the model, the result is really exciting. However, there seems some "minor faults" in some model results. for example, one of my test audio has various repetition words & filler words since it is just a free talk, but the output just ignore them and only reserve one word. 
I mean, yeah, this is helpful in many cases, but sometimes i need a precise output just exactly follows the origin audio, so do we have some options to manually control this case? thanks in advance.

**Comment by Tejasvidash** [comment]
Hi can you please share the code?


Thanks in advance

**Comment by temberature** [comment]
https://huggingface.co/spaces/openai/whisper/discussions/30

**Comment by LaurinmyReha** [comment]
https://github.com/nyrahealth/CrisperWhisper this will solve your problem :)
